### ISSUE TITLE
Support AWS CodeArtifact for Maven

### ISSUE THREAD

**Issue by aripalo** [cdklabs/publib#10]
- Support AWS CodeArtifact for Maven.
- Add documentation and/or example how to configure AWS CodeArtifact for `jsii-release`

### Use Case

I want to publish my CDK constructs built with JSII (NPM/PyPI/Maven) to AWS CodeArtifact - and I've now succesfully done it with NPM+PyPI 👍 

The maven setup seems quite complex so I haven't (at least yet) tried to figure it out myself because my very limited knowledge of Maven and Java-ecosystem ☹️ But I can try some day!

### Proposed Solution

Due to above reason I can't be sure, but to me it seems that the following hardcoded values in `bin/jsii-release-maven` will most probably prevent publishing to CodeArtifact:
- `-DrepositoryId=maven-central` on [ line `133`](https://github.com/aws/jsii-release/blob/master/bin/jsii-release-maven#L133)
- `-DserverId=ossrh` on [line `153`](https://github.com/aws/jsii-release/blob/master/bin/jsii-release-maven#L153)

Basically the solution should be similar as with [npm](https://github.com/aws/jsii-release#npm) and with [pypi (see my PR)](https://github.com/aws/jsii-release/pull/9): Rely on environment variables (which work well in CI-environment).

### Other

* [x] :wave: I may be able to implement this feature request
* [ ] :warning: This feature might incur a breaking change

---

This is a :rocket: Feature Request
